### PR TITLE
JSAEM2-27 fixed: bug in date picker

### PR DESCRIPTION
**Pull request by MichaelCTH**

### CODE REVIEW

File: src/TransCreator/DateSelector.jsx [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
-        textStyle={{ color: '#5C6BC0' }}
-        placeHolderTextStyle={{ color: '#5C6BC0' }}
-        onDateChange={(newDate) => setTransDate(moment(newDate).unix())}
-        disabled={false}
-      />
-    </View>
-  </>
-);
+const DateSelector = ({ transDate, setTransDate }) => {
+  const [datePickerShow, setDatePickerShow] = useState(false);
```
Comment (ikarasz) [inline src/TransCreator/DateSelector.jsx]: let's call it `isDatePickerVisible`

File: src/TransCreator/DateSelector.jsx [hunk 2]
```diff
@@ -1,37 +1,39 @@
...
+  return (
+    <>
+      <MaterialIcons name="date-range" style={styles.dateItemIcon} />
+      <Text>Date: </Text>
+      <View style={styles.dateView}>
+        <Text
```
Comment (ikarasz) [inline src/TransCreator/DateSelector.jsx]: cannot we use `native-base`'s `Text` component

File: src/TransCreator/DateSelector.jsx [hunk 3]
```diff
@@ -1,37 +1,39 @@
...
+          {moment.unix(transDate).format('MM/DD/YYYY')}
+        </Text>
+        <DateTimePicker
+          isVisible={datePickerShow}
+          mode="date"
+          date={new Date(moment.unix(transDate).format('YYYY-MM-DD'))}
+          onCancel={() => setDatePickerShow(false)}
+          onConfirm={(date) => {
+            setTransDate(moment(date).unix());
+            setDatePickerShow(false);
```
Comment (ikarasz) [inline src/TransCreator/DateSelector.jsx]: `setDatePickerVisibility`

File: src/TransCreator/DateSelector.jsx (hunk 1 above)
Comment (MichaelCTH) [reply src/TransCreator/DateSelector.jsx]: fixed

File: src/TransCreator/DateSelector.jsx (hunk 3 above)
Comment (MichaelCTH) [reply src/TransCreator/DateSelector.jsx]: fixed

File: src/TransCreator/DateSelector.jsx (hunk 2 above)
Comment (MichaelCTH) [reply src/TransCreator/DateSelector.jsx]: We are not goIng to use native-base any more, all the previous implementations should get rid of native-base components little by little. Thus, I used Text from 'react-native' here.

Comment (ikarasz) [reply src/TransCreator/DateSelector.jsx]: @MichaelCTH then please create a story for this in jira

**Pull request merged**